src/site/asn1.rst

```diff
@@ -9,10 +9,37 @@ Assignment #1: Playing with Objects is Still Fun
 Task
 ====
 
+You will be creating a simple collection to keep track of ``Country`` objects. This will require you to write classes,
+use the classes, and write some various methods outlined below.
+
+Effectively, the idea is to recreate your assignment 4 from CSCI 161, but in Java. Be aware that the assignment is not
+identical and the requirements outlined here will be what is required.
+
+You will
+
+* Write a ``Country`` class with a few simple methods
+* Write a ``CountryCatalogue`` class with methods making use of instances of a ``Country``
+* Testing your implementation
+
 
 Provided Files
 ==============
 
+You are provided with
+
+* A nearly empty *Country.java* file where you will write your ``Country`` class
+* A nearly empty *CountryCatalogue.java* file where you will write your ''CountryCatalogue`` class
+* An *asn1.java* file with a ``main`` method and some testing code
+* Data files
+    * A *data.txt* file containing some country details
+    * A *continent.txt* file containing country and continent information
```

Comment:
An [intellij project](https://www.jetbrains.com/help/idea/import-project-or-module-wizard.html) would be helpful, maybe. Could set up the tests as tests, instead of a `main` method.

Again, depends if you want unit tests for the first assignment, or if that'll be an assignment two jam.

Reply:
This is my ultimate goal, but as of now, it's a _get this out the door now_ sitch given constraints.